### PR TITLE
enable ThreadId for when testing

### PR DESCRIPTION
**Pull request by soulomoon**
It seems we do have it already, I enable it for default when testing.

### CODE REVIEW

File: hls-test-utils/src/Test/Hls.hs [hunk 1]
```diff
@@ -352,7 +353,7 @@ pluginTestRecorder = do
 -- See 'runSessionWithServer'' for details.
 initialiseTestRecorder :: Pretty a => [String] -> IO (Recorder (WithPriority a), WithPriority (Doc ann) -> IO ())
 initialiseTestRecorder envVars = do
-    docWithPriorityRecorder <- makeDefaultStderrRecorder Nothing
+    docWithPriorityRecorder <- makeDefaultStderrRecorder (Just [ThreadIdColumn])
```
Comment (fendor) [inline hls-test-utils/src/Test/Hls.hs:356]: You probably want to expose `defaultLoggingColumns` and write 

```suggestion
    docWithPriorityRecorder <- makeDefaultStderrRecorder (Just $ [ThreadIdColumn] <> defaultLoggingColumns)
```

**Review by michaelpj — APPROVED** [review]

File: hls-test-utils/src/Test/Hls.hs [hunk 2]
```diff
@@ -348,7 +349,7 @@ pluginTestRecorder = do
 -- See 'runSessionWithServer'' for details.
 initialiseTestRecorder :: Pretty a => [String] -> IO (Recorder (WithPriority a))
 initialiseTestRecorder envVars = do
-    docWithPriorityRecorder <- makeDefaultStderrRecorder Nothing
+    docWithPriorityRecorder <- makeDefaultStderrRecorder (Just $ [ThreadIdColumn] <> defaultLoggingColumns)
```
Comment (jhrcek) [inline hls-test-utils/src/Test/Hls.hs:352]: Just an ocd nitpick :slightly_smiling_face: 
```suggestion
    docWithPriorityRecorder <- makeDefaultStderrRecorder (Just $ ThreadIdColumn : defaultLoggingColumns)
```

Comment (soulomoon) [reply hls-test-utils/src/Test/Hls.hs:352]: Thanks ~

**Comment by michaelpj** [comment]
Build failure seems genuine

**Pull request merged**